mkosi/distributions/rhel_ubi.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import urllib.parse
+from typing import Generator
+
+from mkosi.config import MkosiConfig
+from mkosi.distributions import centos
+from mkosi.installer.dnf import Repo
+
+
+def join_mirror(config: MkosiConfig, link: str) -> str:
+    assert config.mirror is not None
+    return urllib.parse.urljoin(config.mirror, link)
```

Comment:
This is identical to the same function in centos.py, so let's just use that one instead.